packages/expo-localization/src/Localization.types.ts

```diff
@@ -64,4 +64,107 @@ export type Localization = {
    * @example `'America/Los_Angeles'`
    */
   timezone: string;
+  getLocales: () => Locale[];
+  getCalendars: () => Calendar[];
+};
+
+export type Locale = {
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code. Example: `en-US`, "es-419", "pl-PL".
```

Comment:
Let's be consistent with quotes/backticks 😄 Same problem appears in a few other properties below.
```suggestion
   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code. Example: `en-US`, `es-419`, `pl-PL`.
```